lib/recurly.js

```diff
@@ -103,6 +103,8 @@ Recurly.prototype.configure = function (options) {
     this.config.currency = options.currency;
   }
 
+  this.config.extraRequiredFields = options.extraRequiredFields || [];
```

Comment:
I'd like to make this property a bit more succinct. Perhaps simply `required`. It's not always the case that form fields are being passed in, so I think a generic name is best.